stack.yaml

```diff
@@ -1,9 +1,7 @@
-resolver: lts-13.26
+resolver: nightly-2019-09-30
 
 extra-deps:
-  - aeson-1.4.4.0
+  - dependent-map-0.2.4.0
   - dependent-sum-0.5
-  - hedgehog-1.0
-  - primitive-0.7.0.0
-  - tasty-hedgehog-1.0.0.1
-  - time-compat-1.9.2.2
+
+allow-newer: true
```

Comment:
Do you remember why this `allow-newer` is needed?

Reply:
Actually, it's not needed. I had put in it when trying to resolve the correct versions of dependencies and forgot to remove it ;)